### PR TITLE
help page: updated eous/company call to v2 

### PR DESCRIPTION
**Pull request by madhavmansuriya40**

### CODE REVIEW

File: src/app/fyle/help/help.page.ts [hunk 1]
```diff
@@ -24,23 +25,30 @@ export class HelpPage implements OnInit {
...
 
   openContactSupportDialog() {
     this.contactSupportLoading = true;
     from(this.loaderService.showLoader('Please wait', 10000)).pipe(
       switchMap(() => {
-        return this.orgUserService.getAllCompanyEouc();
+        return this.authService.getEou();
+      }),
+      switchMap(eou => {
+        return this.orgUserService.getEmployeesByParams({
```
Comment (Julias0) [inline src/app/fyle/help/help.page.ts:39]: do this inside the service

**Review by iamtarun1993 — APPROVED** [review]

File: src/app/core/services/org-user.service.ts [hunk 2]
```diff
@@ -78,6 +80,10 @@ export class OrgUserService {
     );
   }
 
+  getEmployeesByParams(params) {
+    return this.apiV2Service.get('/employees', {params});
```
Comment (abhishek1234321) [inline src/app/core/services/org-user.service.ts:84]: should we use this or should we use the `search` API, please check with @neeraj once

Comment (madhavmansuriya40) [reply src/app/core/services/org-user.service.ts:84]: here we just need 5 admin users to show in the help dialogue
at other places where to add approver or modify approver functionality is required, we can use search API

Comment (abhishek1234321) [reply src/app/core/services/org-user.service.ts:84]: ok go ahead and merge it in

**Pull request closed without merging**